hugegraph-dist/pom.xml

```diff
@@ -212,6 +229,19 @@
                         </configuration>
                     </execution>
                 </executions>
+                <dependencies>
+                    <dependency>
+                        <groupId>ant-contrib</groupId>
+                        <artifactId>ant-contrib</artifactId>
+                        <version>1.0b3</version>
+                        <exclusions>
+                            <exclusion>
+                                <groupId>ant</groupId>
+                                <artifactId>ant</artifactId>
+                            </exclusion>
+                        </exclusions>
```

Comment:
we still need it?

Reply:
To avoid downloading swagger-ui on non-Linux platforms, the utilization of the `if` tag from `ant-contrib` is demonstrated here.

An alternative [approach](https://maven.apache.org/guides/introduction/introduction-to-profiles.html#OS) could involve controlling profile activation through the `os` attribute. **However, it seems tricky to activate the same profile for both Mac and Linux simultaneously.**

**In order to reuse the configuration of the `maven-antrun-plugin`**, I'm considering setting up the `download-swagger-ui` configuration in the root pom's `pluginManagement`. In This way, it can be reused in `hugegraph-dist`, see [`1534605` (#2277)](https://github.com/apache/incubator-hugegraph/pull/2277/commits/1534605ae8ab64a2d601fd3930ec19967bf8cd27).

Reply:
@imbajin By the way, what does the subtask imply here? Is there a need to support downloading swagger-ui on Windows in THIS PR?

![image](https://github.com/apache/incubator-hugegraph/assets/79143929/53f753ea-fe00-4385-8ad6-d4b4296e1849)

Reply:
no need，fine to skip download in windows

Reply:
> To avoid downloading swagger-ui on non-Linux platforms, the utilization of the `if` tag from `ant-contrib` is demonstrated here.
> 
> An alternative [approach](https://maven.apache.org/guides/introduction/introduction-to-profiles.html#OS) could involve controlling profile activation through the `os` attribute. **However, it seems tricky to activate the same profile for both Mac and Linux simultaneously.**
> 
> **In order to reuse the configuration of the `maven-antrun-plugin`**, I'm considering setting up the `download-swagger-ui` configuration in the root pom's `pluginManagement`. In This way, it can be reused in `hugegraph-dist`, see [`1534605` (#2277)](https://github.com/apache/incubator-hugegraph/pull/2277/commits/1534605ae8ab64a2d601fd3930ec19967bf8cd27).
> 

nice refactor （try it）

Reply:
> > To avoid downloading swagger-ui on non-Linux platforms, the utilization of the `if` tag from `ant-contrib` is demonstrated here.
> > An alternative [approach](https://maven.apache.org/guides/introduction/introduction-to-profiles.html#OS) could involve controlling profile activation through the `os` attribute. **However, it seems tricky to activate the same profile for both Mac and Linux simultaneously.**
> > **In order to reuse the configuration of the `maven-antrun-plugin`**, I'm considering setting up the `download-swagger-ui` configuration in the root pom's `pluginManagement`. In This way, it can be reused in `hugegraph-dist`, see [`1534605` (#2277)](https://github.com/apache/incubator-hugegraph/pull/2277/commits/1534605ae8ab64a2d601fd3930ec19967bf8cd27).
> 
> nice refactor （try it）

done in [`1534605` (#2277)](https://github.com/apache/incubator-hugegraph/pull/2277/commits/1534605ae8ab64a2d601fd3930ec19967bf8cd27)